### DISCUSSION TITLE
Fascinated by this build, more parts arriving soon.

### DISCUSSION THREAD

**Discussion by ej012345** [picocomputer/community#5] [Q&A]
Component placement subject to change once I begin wiring.
![ej pico computer](https://user-images.githubusercontent.com/131208326/236048488-9a542f3f-9352-4667-9ef1-9c77cbbd1938.jpg)
EJ

**Comment by rumbledethumps** [comment]
Welcome aboard. You can get rp6502-vscode working on your dev system while you wait for parts.

**Comment by ej012345** [comment]
Removed the extra bb power rails... got vscode working on Linux Mint... hooked up secondary pico to pico... wiring, wiring, wiring.

**Comment by ej012345** [comment]
Alrighty then, let's try this again. Board one was a learning experience.
![ej pico computerV0 2](https://github.com/picocomputer/discussions/assets/131208326/a5fdbcec-36ee-4ccf-b2ce-a27780b78864)

**Comment by rumbledethumps** [comment]
There's no ideal layout. Just go for it. Those unassuming 74XXs are the most complicated part of the build.

**Comment by ej012345** [comment]
![IMG_2450](https://github.com/picocomputer/discussions/assets/131208326/55b16a31-a6d2-4487-9dd9-f899100f32bc)
Progress is being made... Pin numbers are mostly for Pico-2 as it is upside down.
![ej pico computerV0 5](https://github.com/picocomputer/discussions/assets/131208326/fca90ec2-8351-4f61-9c20-754fc03e1486)

**Comment by ej012345** [comment]
![IMG_2550](https://github.com/picocomputer/discussions/assets/131208326/489ac3ec-4c0a-4401-801c-983aed8f4e94)
![IMG_2569](https://github.com/picocomputer/discussions/assets/131208326/1b8c34b2-afaf-4186-84a8-ae91872488e4)

**Comment by rumbledethumps** [comment]
Congratulations.
